scripts/griddle.jsx

```diff
@@ -649,7 +650,7 @@ var Griddle = React.createClass({
         ) : "";
     },
     getCustomGridSection: function(){
-        return <this.props.customGridComponent data={this.props.results} className={this.props.customGridComponentClassName} />
+        return <this.props.customGridComponent data={this.props.results} className={this.props.customGridComponentClassName} {... this.props.gridMetadata} />
```

Comment:
It's spread operator. I should remove that space there.

I don't use `this.props.gridMetadata` because then what's the point of passing other properties like `rowClick`. You have to create another property inside the `gridMetadata` that simulate the same result. When we customize the `grid` we change everything, in my opinion, we should have all the properties for the `regular grid` + whatever you need in the `gridMetadata` but that's not a big deal we can change it.